src/server/api/general.js

```diff
@@ -290,41 +290,35 @@ export default function useGeneralApi(app) {
         try {
             if (!emailRegex.test(email.toLowerCase()))
                 throw new Error('not valid email: ' + email);
-            const existingUser = yield findUser({
+            let user = yield findUser({
```

Comment:
I think we should skip the lookup entirely, unless we are sure that the existing user instance won’t lead to an unrecoverable account

Reply:
i'll change it to an upsert

Reply:
actually should we remove any rows with matching usernames, and then create a new row for this approved user?

Reply:
should we remove rows w/ matching usernames or emails, or just usernames?

Reply:
I was thinking we just add more users, or is there an unique index?

Reply:
no unique index; i changed it to just make a new user record (see commit "just make a new user record" below;))